test/query_test.exs

```diff
@@ -677,6 +677,55 @@ defmodule QueryTest do
     assert [[neg_inf]] == query("SELECT $1::numeric", [neg_inf])
   end
 
+  @tag max_pg_version: "14.0"
```

Comment:
If these tests are merged, I was debating what the tag should be because we need it to be exclusive. This is less verbose but could be confusing because of the min tag behaviour. I could also make it `max_pg_version_exclusive` or something similar.